project/config/liofa/config/liofa_pledges.countsettings.yml

```diff
@@ -1,5 +1,5 @@
 pledge_count_submissions: '26361'
 pledge_count_offset: '0'
-pledge_count_total: 35454
+pledge_count_total: 37139
```

Comment:
Should this not be stored in the db and not config or is the config updated on the fly?

Reply:
It is in config ignore so it will never be imported.